### PR TITLE
Fix session ID generation

### PR DESCRIPTION
**Pull request by at-wat**
Use properly seeded mathmatical random generator.

### Reference issue
Related: https://github.com/pion/ice/issues/203

### CODE REVIEW

File: util.go [hunk 1]
```diff
@@ -50,8 +52,7 @@ func (t ConnectionRole) String() string {
 }
 
 func newSessionID() uint64 {
-	r := rand.New(rand.NewSource(time.Now().UnixNano()))
-	return uint64(r.Uint32()*2) >> 2
+	return globalMathRandomGenerator.Uint64()
```
Comment (at-wat) [inline util.go:55]: In RFC, session ID is recommended to be crypto random.
Adding `randutil.CryptoUint64` https://github.com/pion/randutil/pull/2

Comment (at-wat) [reply util.go:55]: Using crypto rand in `NewJSEPSessionDescription` should make it returning error. It requires API change.

File: jsep.go [hunk 2]
```diff
@@ -51,11 +51,15 @@ var extMapURI = map[int]string{
 // NewJSEPSessionDescription creates a new SessionDescription with
 // some settings that are required by the JSEP spec.
 func NewJSEPSessionDescription(identity bool) *SessionDescription {
```
Comment (at-wat) [inline jsep.go:53]: It must break API to return `(*SessionDescription, error)` to use `crypto/rand` in SessionID without panic.
@Sean-Der what do you think?

Comment (Sean-Der) [reply jsep.go:53]: 100% support from me!

I would be fine even doing a new minor version. It isn't a big change, and there is an API difference so users will clearly see.

**Comment by at-wat** [comment]
Merging and adding a new release v2.4.0

**Pull request merged**